test/Tool/more-knownbits11.opt

```diff
@@ -0,0 +1,12 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i8 = var (p)
+%1:i8 = var (z)
+%2:i8 = udivexact %0, %1
+%3:i8 = ctpop %2
+%4:i1 = eq %3, 1:i8
+cand %4 1:i1
```

Comment:
I added the 'z' flag just to make sure that denominator of division operation is never zero. The intention was basically to test power of two feature only. I have fixed the test case by removing 'z' flag. Similarly in other test case too.